prompt/prompt-executor/prompt-executor-clients/prompt-executor-ollama-client/src/commonMain/kotlin/ai/koog/prompt/executor/ollama/client/OllamaClient.kt

```diff
@@ -181,9 +183,13 @@ public class OllamaClient(
         if (response.status.isSuccess()) {
             return parseResponse(response.body<OllamaChatResponseDTO>())
         } else {
-            val errorResponse = response.body<OllamaErrorResponseDTO>()
-            logger.error { "Ollama error: ${errorResponse.error}" }
-            throw RuntimeException("Ollama API error: ${errorResponse.error}")
+            val exception = LLMClientException(
+                clientName = clientName,
+                statusCode = response.status.value,
+                errorBody = response.bodyAsText(),
+            )
+            logger.error { exception.message }
```

Comment:
Please also provide the exception itself as a parameter to `logger.error(e) { ... }`